scripts/Mend/Mend-Scan.ps1

```diff
@@ -0,0 +1,14 @@
+function Get-Version {
```

Comment:
I did the rename in a separate commit to track the history, but GitHub didn't pick it up and shows it as delete/added :/